server/src/controllers/workedHours.controller.js

```diff
@@ -0,0 +1,53 @@
+const CrudRestController = require('./crud-rest.controller');
+const WorkedHoursRepository = require('../repositories/workedHours.repository')
+
+/**
+ * Base Controller
+ */
+class WorkedHoursController extends CrudRestController {
```

Comment:
This controller will have only two endpoints it should extend from BaseRestController